client/src/component/pageHeader/pageHeader.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+import "./pageHeader.css";
+
+class pageHeader extends Component {
```

Comment:
In the future try to name the components following the description of the ticket. I think you named it `Header` before while the PR asked you to create `PageHeader`